src/wx/visualboyadvance-m.metainfo.xml

```diff
@@ -20,15 +17,17 @@
   </p>
  </description>
  <screenshots>
-  <screenshot type="default">https://visualboyadvance-m.org/screenshots/1.png</screenshot>
-  <screenshot>https://visualboyadvance-m.org/screenshots/2.png</screenshot>
-  <screenshot>https://visualboyadvance-m.org/screenshots/3.png</screenshot>
-  <screenshot>https://visualboyadvance-m.org/screenshots/4.png</screenshot>
+     <screenshot type="default"><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/visual-boy-advance-installed.webp</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/visualboyadvance-ubuntu-software-store.webp</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/screen-filters-vba.webp</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/enable-cheats-on-visual-boy-advance.png</image></screenshot>
+     <screenshot><image type="source">https://i0.wp.com/visualboyadvance.org/wp-content/uploads/2023/01/visualboyadvance-on-android.webp</image></screenshot>
  </screenshots>
+ <launchable type="desktop-id">visualboyadvance-m.desktop</launchable>
  <releases>
-   <release version="2.1.9" date="2024-02-03"/>
+   <release version="2.1.8" date="2023-12-14"/>
  </releases>
- <url type="homepage">http://visualboyadvance-m.org/</url>
+ <url type="homepage">https://visualboyadvance.org</url>
```

Comment:
Wrong ALL wrong, visualboyadvance.org isn't our domain, they steal our releases and host roms with it. visualboyadvance-m.org is our domain.

Reply:
Sorry about this! Once again, at the time the website seemed to be down....
I will revert it.

Reply:
Reverted.